<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgBulkRegister</class>
 <widget class="QDialog" name="CDlgBulkRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>卡批量注册</string>
  </property>
  <property name="statusTip">
   <string>%1卡批量注册</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(D:/WinParkUI/debug/Image/NewIcon/CommonMiddleBG-normal.jpg);</string>
  </property>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>5</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url( );
color: rgb(85, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblClose">
   <property name="geometry">
    <rect>
     <x>689</x>
     <y>5</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabRecord">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>91</y>
     <width>631</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(none);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtCardID">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(none);</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>289</x>
     <y>620</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="styleSheet">
       <string notr="true">background-image: url(none);</string>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnClose">
      <property name="styleSheet">
       <string notr="true">background-image: url(none);</string>
      </property>
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>44</y>
     <width>641</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QCheckBox" name="chkID">
      <property name="styleSheet">
       <string notr="true">background-image: url(none);</string>
      </property>
      <property name="text">
       <string>自动产生自编号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">background-image: url(none);</string>
      </property>
      <property name="text">
       <string>开始编号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spValue">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(none);</string>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">background-image: url(none);</string>
      </property>
      <property name="text">
       <string>自编号位数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spDigit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(none);</string>
      </property>
      <property name="minimum">
       <number>4</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">background-image: url(none);</string>
      </property>
      <property name="text">
       <string>批量车型</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbxCarType">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(none);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">background-image: url(none);</string>
      </property>
      <property name="text">
       <string>批量有效期(月)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spMonth">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(none);</string>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>1200</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>72</y>
     <width>641</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(none);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="chkEnterCard">
       <property name="whatsThis">
        <string>1</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(none);</string>
       </property>
       <property name="text">
        <string>入场必须刷卡</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkLeaveCard">
       <property name="whatsThis">
        <string>2</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(none);</string>
       </property>
       <property name="text">
        <string>出场必须刷卡</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkMIO">
       <property name="whatsThis">
        <string>3</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(none);</string>
       </property>
       <property name="text">
        <string>多进多出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>edtCardID</zorder>
  <zorder>lblTitle</zorder>
  <zorder>lblClose</zorder>
  <zorder>tabRecord</zorder>
  <zorder>frame</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
